File: py/tests/testRemoveVowelsFromStr/test_RemoveVowelsFromStr.py

import unittest
from parameterized import parameterized as p
from solns.removeVowelsFromStr.removeVowelsFromStr import *

class UnitTest_RemoveVowelsFromStr(unittest.TestCase):
    @p.expand([
        ["leetcodeisacommunityforcoders","ltcdscmmntyfrcdrs"],
        ["aeiou",""],["",""]
    ])
    def test_naive(self,s,expected):
        self.assertEqual(Solution.regex(s),expected)

File: cs/sample/Sample.Tests/RemoveVowelsFromStr/testRemoveVowelsFromStr.cs
using System;
using Xunit;
using Sample.RemoveVowelsFromStr;

namespace Sample.Tests{
    public class UnitTest_RemoveVowelsFromStr{
        [Theory]
        [InlineData("aeiou","")]
        [InlineData("leetcodeisacommunityforcoders","ltcdscmmntyfrcdrs")]
        public void checkResult(string s,string expected){
            Assert.Equal(expected,removeVowelsFromStr.naive(s));
        }
    }
}
